macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -24,7 +23,7 @@
         {%- else %}
             true
         {%- endif %}
-        {% if where_expression %} and {{ where_expression }} {% endif %}
+        {% if metric_properties.where_expression != "Null" %} and {{ metric_properties.where_expression }} {% endif %}
```

Comment:
This is no longer valid.
Let's remove references for `Null`.

Reply:
Good Catch